### PR TITLE
Add FAMC.STAT UNCHALLENGED

### PR DESCRIPTION
**Pull request by tychonievich**
As discussed in #195

### CODE REVIEW

File: specification/gedcom-03-datamodel.md [hunk 1]
```diff
@@ -2971,6 +2971,7 @@ are not currently given their own tag. Cultural or personal gender preference ma
 | `CHALLENGED` | Linking this child to this family is suspect, but the linkage has been neither proven nor disproven. |
 | `DISPROVEN` | There has been a claim by some that this child belongs to this family, but the linkage has been disproven. |
 | `PROVEN` | There has been a claim by some that this child does not belong to this family, but the linkage has been proven. |
+| `UNCHALLENGED` | No reason to suspect the link between this child and this family has been identified. `UNCHALLENGED` is implied if this structure is not present. |
```
Comment (dthaler) [inline specification/gedcom-03-datamodel.md:2974]: no reason to suspect it's been identified sounds like it's in doubt, is that what is meant?  I might suggest

```suggestion
| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. `UNCHALLENGED` is implied if this structure is not present. |
```

I would also ask about the stated meaning of PROVEN.   If the linkage has been proven, but there has _not_ been a claim that this child does not belong to this family, what status should be used?

Comment (tychonievich) [reply specification/gedcom-03-datamodel.md]: `PROVEN` was added with this wording in 5.5.1; not sure what the rationale was

Comment (dthaler) [reply specification/gedcom-03-datamodel.md]: Tracking via issue #225 and proposed PR #228

**Review by dthaler — APPROVED** [review]

**Comment by tychonievich** [comment]
Discussed 2022-10-18
"Unchallenged" and "unknown" are not the same idea, so this is not the correct solution yet. We might be able to come up with some kind of "none of the above" value (perhaps `OTHER`), or might need to add two values.

File: specification/gedcom-03-datamodel.md [hunk 2]
```diff
@@ -2971,6 +2971,7 @@ are not currently given their own tag. Cultural or personal gender preference ma
 | `CHALLENGED` | Linking this child to this family is suspect, but the linkage has been neither proven nor disproven. |
 | `DISPROVEN` | There has been a claim by some that this child belongs to this family, but the linkage has been disproven. |
 | `PROVEN` | There has been a claim by some that this child does not belong to this family, but the linkage has been proven. |
+| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. `UNCHALLENGED` is implied if this structure is not present. |
```
Comment (tychonievich) [inline specification/gedcom-03-datamodel.md:2974]:
```suggestion
| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. |
```

Remove the "if not present" clause due to comments in pull request conversation thread

**Comment by dthaler** [comment]
@tychonievich needs rebase to resolve conflict

**Comment by tychonievich** [comment]
See #195 for why this is being closed

**Pull request closed without merging**